### ISSUE TITLE
[BUG] Steam client Small Mode & Big Mode

### ISSUE THREAD

**Issue by Gawnz1** [ValveSoftware/steam-for-linux#11323]
#### Your system information

* Steam client version: 
Steam Beta Branch:  Stable Client
Steam Version:  1726604483
Steam Client Build Date:  Tue, Sep 17 11:03 PM UTC -08:00
Steam Web Build Date:  Tue, Sep 17 2:12 AM UTC -08:00
Steam API Version:  SteamClient021
* Distribution (e.g. Ubuntu): Debian 12 with latest updates.
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes, fully updated.
* GPU: - Nvidia with latest drivers.

#### Please describe your issue in as much detail as possible:
When you switch to Small mode and press X to close the window or the small library, the window can't be closed and re-appears again.
After switching to Big Mode, same thing is happening.
Basically, Small mode is bugged and when once activated, the Big Mode also becomes bugged after switching again from Small mode.

#### Steps for reproducing this issue:

1. Click on View -> Small Mode
2. Try to click X to close the window (which is in small mode) and it will re-appear again.
3. Try to switch again from Small mode to Big Mode and same will happen.
